<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QBJobViewUi</class>
 <widget class="QWidget" name="QBJobViewUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jobs</string>
  </property>
  <layout class="QHBoxLayout">
   <item>
    <widget class="Q3GroupBox" name="jobBox">
     <property name="title">
      <string>Jobs</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QBJobListView" name="jobList" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QPushButton" name="dequeueButton">
       <property name="text">
        <string>Dequeue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="executeButton">
       <property name="text">
        <string>Execute</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>241</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QBJobListView</class>
   <extends>QWidget</extends>
   <header location="global">q4banking/qbjoblist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
